<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MiniMazeGameWidgetIII</class>
 <widget class="QWidget" name="MiniMazeGameWidgetIII">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MiniMazeGameWidgetIII</string>
  </property>
  <widget class="QLabel" name="scrollLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>150</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>fill in with scroll</string>
   </property>
  </widget>
  <widget class="QLabel" name="NPCLabel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>190</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>NPCLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="IntroLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>IntroLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="end_page_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>170</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>end_page_label</string>
   </property>
  </widget>
  <widget class="QLabel" name="PlayerLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Player</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="end_words_label">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
